portfolio/src/main/webapp/script.js

```diff
@@ -6,15 +6,24 @@ function changeImage() {
 }
 
 /**
-  * Creates and returns an HTML list element containing
-  * a string provided by the the text parameter.
-  * @param {String} text Content of the list element.
-  * @return {<li>} HTML list element with the contents of the text parameter.
+  * Creates and adds a post card containing a caption and 
+  * a score as provided by the post parameter.
+  * @param {post} A user defined object with keys: (String)comment,
+  * (String)username, and (int)score. 
   */
-function createListElement(text) {
-  const listElement = document.createElement('li');
-  listElement.innerText = text;
-  return listElement;
+function createListElement(post) {
+  var card = '';
+  card += '  <div class="card m-1">';
+  card += '    <div class="card-body">';
+  card += '      <p class="card-text">' + post.comment + '</p>';
+  card += '      <div class="btn-group" role="group">';
+  card += '        <button onclick="votePost(this, -1)" type="button" class="btn btn-sm btn-primary"> - </button>';
+  card += '        <button type="button" id="score" class="btn btn-sm btn-outline-primary disabled"> ' + post.score + ' </button>';
+  card += '        <button onclick="votePost(this, 1)" type="button" class="btn btn-sm btn-primary"> + </button>';
+  card += '      </div>';
+  card += '    </div>';
+  card += '  </div>';
+  document.getElementById('posts').innerHTML += card;
```

Comment:
I know this is more compact but it's better to use the ```document.createElement``` API for creating these elements. It's difficult to debug things like syntax errors with this approach.